test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -599,6 +676,11 @@ def startTest(self, test):
             self.__failed_tc_count_msg = True
             raise TCThresholdReached
         timeout = self.__get_timeout(test)
+        param_dict = self.__get_param_dictionary()
```

Comment:
Do we need to populate this here? I think we can populate this in `configure` only since it doesn't depend on test which is about to start